lib/address.js

```diff
@@ -116,6 +115,10 @@ Address.prototype._classifyArguments = function(data, network, type) {
 Address.PayToPublicKeyHash = 'pubkeyhash';
 /** @static */
 Address.PayToScriptHash = 'scripthash';
+/** @static */
+Address.PayToPublicKeyHashReplay = 'pubkeyhashreplay';
+/** @static */
+Address.ScriptHash = 'scripthash';
```

Comment:
looking at the code the Address.ScriptHash  appears to me to be the pay to script hash with replay protection, so to be coherent with the previous ones I would rename it to:
Address.PayToScriptHashReplay= 'scripthashreplay';
please check all the  "if"  using it and change accordingly
Also why we have used the same constant value of standard PayToScriptHash? seems better to discriminate it, and if in some places we need not handle it in different ways for backward compatibility add an if clause

Reply:
I have created a ticket to do these changes after zendoo launch